project-management/collateral.md

```diff
@@ -104,3 +105,38 @@ Here's an example FreshBooks project for a scope item in the CalFWD project:
 ![CalFWD project in FreshBooks](../images/freshbooks-calfwd.png)
 
 For further reference, FreshBooks maintains [its own documentation on how projects work](https://www.freshbooks.com/support/which-billing-method-should-i-choose).
+
+## Research
```

Comment:
Should we include something that ties this into the item in our scope of work template designated for this kind of research? That would provide guidance and bounds around how much time to spend on it if the project has budgeted time for it

Reply:
Something more specific than this?

> Instead, most projects include hours specifically earmarked for research, so staff are able to acquire and/or deepen expertise prior to implementation.

Reply:
At this point, its not most projects that will have hours this (though we're trying to do it more). Maybe we can just make a note to check the scope for the project you're working on to see if it includes a research component.

Reply:
Done, @derekeder, and glad to have research baked into scoping moving forward. Even an hour or two is beneficial.